modules/postgres/postgres_test.go

```diff
@@ -288,6 +288,44 @@ func TestWithInitScript(t *testing.T) {
 	require.NotNil(t, result)
 }
 
+func TestWithOrderedInitScript(t *testing.T) {
+	ctx := context.Background()
+
+	ctr, err := postgres.Run(ctx,
+		"postgres:15.2-alpine",
+		// Executes first the init-user-db shell-script, then the insert-user SQL script
+		postgres.WithOrderedInitScripts(
+			filepath.Join("testdata", "init-user-db.sh"),
+			filepath.Join("testdata", "insert-user.sql"),
```

Comment:
suggestion: because `init-user-db` < `insert-user`, it will be applied first even without the changes in the PR.

I'd simply make sure the logic in this PR is satisfy, probably checking the files are renamed in the target dir of the container 🤔  Thoughts?

Reply:
I'm happy to do that, but I couldnt find an easy way to list the files that are inside the container after it's created.

What do you think of just renaming `insert-user.sql` to `do-insert-user.sql` ?

Reply:
That would work, just making sure the insert file has a name that goes first (i.e. `aaaa-insert-user.sql` for clearness) and if the database is there it means the files where processed correctly.

For checking the content of the directory, you could `ctr.Exec(ctx, []string{"ls -l /path/to/files"})`, get&read the reader output, and verify the names included the new prefixes

Reply:
Done. Thanks for the guidance :)